lib/code_corps/github/sync/sync.ex

```diff
@@ -99,18 +101,53 @@ defmodule CodeCorps.GitHub.Sync do
     |> transact()
   end
 
-  def sync_issues(repo) do
+  def sync_repo(repo) do
+    {:ok, pulls} = GitHub.API.Repository.pulls(repo)
+    github_pull_requests = Enum.map(pulls, &sync_pull(&1, repo))
+    results = sync_issues(repo, github_pull_requests)
+    github_issues = for {:ok, %{github_issue: github_issues}} <- results, do: github_issues
+    github_issues = Enum.map(github_issues, fn issue -> {:ok, issue} end)
+    tasks = for {:ok, %{tasks: tasks}} <- results, do: tasks
+    tasks = tasks |> List.flatten
+    tasks = Enum.map(tasks, fn task -> {:ok, task} end)
+    sync_comments(repo, github_issues, tasks)
+    repo
+  end
+
+  def sync_pull(pull, repo) do
+    Multi.new
+    |> Multi.merge(__MODULE__, :return_repo, [repo])
+    |> Multi.merge(GitHub.Sync.PullRequest, :sync, [pull])
+    |> transact()
+  end
+
+  def sync_issues(repo, github_pull_requests) do
     {:ok, issues} = GitHub.API.Repository.issues(repo)
-    Enum.map(issues, &sync_issue(&1, repo))
+    Enum.map(issues, &sync_issue(&1, repo, github_pull_requests))
   end
 
-  def sync_issue(issue, repo) do
+  def sync_issue(issue, repo, github_pull_requests) do
     Multi.new
     |> Multi.merge(__MODULE__, :return_repo, [repo])
+    |> Multi.merge(__MODULE__, :find_pull_request, [issue, github_pull_requests])
     |> Multi.merge(GitHub.Sync.Issue, :sync, [issue])
     |> transact()
   end
 
+  def sync_comments(repo, github_issues, tasks) do
+    {:ok, comments} = GitHub.API.Repository.comments(repo)
+    Enum.map(comments, &sync_comment(&1, repo, github_issues, tasks))
+  end
+
+  def sync_comment(comment, repo, github_issues, tasks) do
+    Multi.new
+    |> Multi.merge(__MODULE__, :return_repo, [repo])
+    |> Multi.merge(__MODULE__, :find_issue, [comment, github_issues])
+    |> Multi.merge(__MODULE__, :find_tasks, [comment, tasks])
+    |> Multi.merge(GitHub.Sync.Comment, :sync, [comment])
```

Comment:
This is the part that appears not to be working. I got everything up until here working.